program-test/src/lib.rs

```diff
@@ -502,6 +502,15 @@ impl Default for ProgramTest {
 }
 
 impl ProgramTest {
+    /// Create a `ProgramTest` with the given arguments, where `program_name` *must* match the
```

Comment:
Looks like the default impl of `ProgramTest` already has a pretty good description: https://docs.rs/solana-program-test/1.7.0/solana_program_test/struct.ProgramTest.html#impl-Default. Do you mind copying that here and then also adding in the snippet from `add_program` which explains how `program_name` will be used to locate the shared object?

Reply:
Sure, I can do that, though I think it would be better to point to their documentation instead of duplicating the comments under `new`? After all, it's just a wrapper around `default()` and `add_program()`. What do you think?

Reply:
Yup, that works, thanks!